pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -472,6 +472,11 @@ public Boolean get() {
             // Register heartbeat and bootstrap namespaces.
             this.nsService.registerBootstrapNamespaces();
 
+            // Register pulsar system namespaces
+            if (config.isTransactionCoordinatorEnabled()) {
+               this.nsService.registerNamespace(NamespaceName.SYSTEM_NAMESPACE.toString(), false);
```

Comment:
We might need to document this. If a user already has a Pulsar cluster (e.g. 2.4.0), the user wants to upgrade to 2.5.0 to enable transaction. He/she has to create a `pulsar/system` namespace and the coordinator topic. So it might be making sense to add a command tool (similar as PulsarClusterMetadataSetup) to upgrade the cluster to enable transaction.

Reply:
Added a new PulsarTransactionCoordinatorMetadataSetup,  use this setup can create pulsar/system namespace and tc assign partitioned topic.

Considering we don't have documentation for transaction now, i would like to create a task to track the document.

Reply:
Here is the document task tracker https://github.com/apache/pulsar/issues/5530